tests/test_qaoa_objective_sk.py

```diff
@@ -0,0 +1,187 @@
+###############################################################################
+# // SPDX-License-Identifier: Apache-2.0
+# // Copyright : JP Morgan Chase & Co
+###############################################################################
+import numpy as np
+import pytest
+from functools import partial
+from qiskit_aer import AerSimulator
+from itertools import combinations
+from qokit.sk import sk_obj, get_sk_terms, get_random_J
+from qokit.maxcut import maxcut_obj
+from qokit.qaoa_objective_sk import get_qaoa_sk_objective
+from qokit.utils import brute_force, precompute_energies
+from qokit.parameter_utils import get_sk_gamma_beta
+from qokit.fur import get_available_simulators, get_available_simulator_names
+
+
+rng = np.random.default_rng(seed=42)
+
+qiskit_backend = AerSimulator(method="statevector")
```

Comment:
`qiskit_backend`, not used anywhere in this file

Reply:
Removed!